snap7/common.py

```diff
@@ -29,9 +29,11 @@ class ADict(dict):
 
 
 class Snap7Library:
-    """
-    Snap7 loader and encapsulator. We make this a singleton to make
-    sure the library is loaded only once.
+    """Snap7 loader and encapsulator. We make this a singleton to make
+        sure the library is loaded only once.
+
+    Attributes:
+        lib_location: full path to the `snap7.dll` file. Defaults to None
```

Comment:
From the type information, one already knows the default value is none. I would add "optional".

Reply:
In case of optionals with a default value?

Reply:
optional implies a default of `None`